spyder/plugins/workingdirectory/tests/test_workingdirectory.py

```diff
@@ -7,14 +7,32 @@
 
 # Test library imports
 import pytest
+import os
+import os.path as osp
 
 # Local imports
 from spyder.plugins.workingdirectory.plugin import WorkingDirectory
+from spyder.config.base import get_home_dir
+from spyder.config.main import CONF
 
 
 @pytest.fixture
 def setup_workingdirectory(qtbot):
-    """Setupp working directory plugin."""
+    """Setup working directory plugin."""
+    workingdirectory = WorkingDirectory(None)
+    qtbot.addWidget(workingdirectory)
+    workingdirectory.show()
+
+    return workingdirectory, qtbot
+
+@pytest.fixture
+def setup_workingdirectory_startup(qtbot):
+    new_wdir = 'new_workingdir/'
```

Comment:
Maybe it is better to define this as a CONSTANT outside so that it can be used later at the 


```
CONSTANT = 'new_workingdir/'
...

assert act_wdir == CONSTANT
```

Reply:
I don't think you need a new fixture here, just adding a pytest marker to modify the current one, as I described it above for the IPython console fixture.